packages/integrations/components/integrationsManager/index.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// User Config.
+import userConfig from '@irvingjs/irving.config';
+// Redux.
+import { connect } from 'react-redux';
+import { actionHydrateComponents } from '../../actions';
+// Integration components.
+import GoogleAnalytics from '../googleAnalytics';
+
+/**
+ * Integrations Manager.
+ *
+ * A component that consumes a `irving/integrations` configuration from the
+ * REST API, Irving config, or persisted Redux state slice and renders the
+ * corresponding integrations on the client.
+ *
+ * @param {Object}   props - All props.
+ * @param {Object}   props.integrations - Config object for active integrations.
+ * @param {boolean}  props.isHydrated - Has the component map already been hydrated?
+ * @param {function} props.hydrateComponents - Redux action for updating the `componentMap` in the state tree.
+ * @param {Object}   props.hydratedComponents - The existing component key/prop pairs.
+ */
+const IntegrationsManager = ({
+  integrations,
+  isHydrated,
+  hydrateComponents,
+  hydratedComponents,
+}) => {
+  // Declare a map of available integration components with a corresponding key.
+  const componentMap = [
+    { key: 'googleAnalytics', el: GoogleAnalytics },
+  ];
```

Comment:
As noted above, I think this would make more sense in another file being imported.